v1/submissions.go

```diff
@@ -250,3 +475,57 @@ func (e *Endpoint) GetSubmissions() ([]Submission, error) {
 
 	return findings, nil
 }
+
+/*
+	GetSubmission gets report/submission by code/id
+*/
+func (e *Endpoint) GetSubmission(code string) (*SubmissionDetails, error) {
+	var submi SubmissionDetails
+	var respBytes []byte
+	var err error
+	var req *http.Request
+	url := apiEndpointV1 + "/SCHIBSTED-LKJ2GIWL"
```

Comment:
This looks hard coded ;)

Reply:
:(
I had to test the case for the following comment...  I'm not trying to explain the mistake, it's my bad, I just want to show the connection between them